webapi/src/main/scala/org/knora/webapi/routing/RouteUtilADM.scala

```diff
@@ -90,6 +89,24 @@ object RouteUtilADM {
     }
   }
 
+  def transformResponseIntoExternalFormat(
+    response: AdminResponse,
```

Comment:
**thought**: To me the `AdminResponse` marker trait  feels like a bit to much code for not a real benefit.

Seeing that this function always returns the `response` as the same type and even unchanged if it is not doing anything with it I am wondering if we really need a marker trait `AdminResponse`. The underlying function could be generic and accept all inputs:

`def transformResponseIntoExternalFormat[A](response: A): Task[A]`

wdyt?

Reply:
I implemented it like this to follow the existing pattern (and because at first it seemed hard to get `SipiIT` to work without a trait. But in the end this really turned out useless, so I'm happy to remove it.